### ISSUE TITLE
[Supporting platform - Make code- New Project]: After setting the viewport to 320*256px, settings icon is overlapping with expand collapsed button.

### ISSUE THREAD

**Issue by BURRAshiva01** [microsoft/pxt-microbit#6119]
### User Experience:
Low vision users who depend on zooming functionality for better readability will be impacted if the content present on the page is not visible or truncated after setting the viewport to 320*256px. They will not be able to access the share the project controls to share the project.

**Note:** User credentials should NOT be included in the bug.

### Prerequisite:
Achieve viewport 320 X 256 by adjusting the browser. (Open developer's tool, resize and zoom the browser to achieve viewport 320 X 256).

### Repro Steps:

1. Launch application URL: https://makecode.microbit.org/.
2. Home page will open and create 'New project'  
3. Navigate "settings" option and select it

### Actual Result:
**Issue:** : After setting the viewport to 320*256px, settings icon is overlapping with expand/collapsed button.

### Expected Result:
After setting the viewport to 320*256px, 'Settings' control should be visible clearly without overlapping or truncation.

### MAS Reference:
[MAS 1.4.10 – Reflow](https://aka.ms/MAS1.4.10)

**ARIA Authoring Ref Link/Fluent Patterns Guidance /HITS Guidance:**
NA

**UIA/DOM details available:** NA

**Note:**: Same issue replicating at 'Simulator' control.

### Reference Links:

- [Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.
- External Bug Process: If this bug belongs to an external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information, please use this [External Bug Process (sharepoint.com)](https://microsoft.sharepoint.com/teams/cea11y/SitePages/External-Bug-Process.aspx?xsdata=MDN8MDF8fDE1MThmM2JlZGFjMDRlZDhiZmViYTUxMDA2M2NiZWI3fDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDF8MHw2Mzc3NDEyNTM2MTEzNzIwNzR8R29vZHxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaUlpd2lRVTRpT2lJaUxDSlhWQ0k2TVRKOQ%3D%3D&sdata=MnlpSXlVMnpGT3cvZ0xJNnFWUWZWTGt6dmo1VkJMZFpIZHBvUUduNDhyMD0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2cv-kgeetala%40microsoft.com). “HCL Staff should not log any third-party external bugs. Should be routed to EDAD team”.
- Please reach out to [C&AI Teams channel](https://apc01.safelinks.protection.outlook.com/?url=https://teams.microsoft.com/l/team/19%253a3cd071821b694f11b054d878453cd75f%2540thread.skype/conversations?groupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=02%7c01%7cAklavyaM%40hcl.com%7c24e89c6ac26f42cfe40c08d7667e49f3%7c189de737c93a4f5a8b686f4ca9941912%7c0%7c0%7c637090564906844600&sdata=22JjBMoA1ID/KnYqFa/OB%2BrkTaLh5YYsGBd/7Bm7u5w%3D&reserved=0) for any process-related queries.

### Test Environment:
**OS Version:** Windows 11 23H2 (OS Build:22631.3007)
**Browser:** Chrome Version 121.0.2277.4 (Official build) dev (64-bit).
**Test Matrix:** Chrome
**URL:** https://makecode.microbit.org/.

### Attachment:

<img width="539" alt="Image" src="https://github.com/user-attachments/assets/7a75a1dd-5908-4d04-b4af-77ed0848c1a5" />
<img width="971" alt="Image" src="https://github.com/user-attachments/assets/33c48134-7665-4b60-a150-c2b3b833dbfb" />

**Comment by Yamini234u** [comment]
As checked issue on https://makecode.microbit.org/#editor. The issue has been fixed. Hence, closing the bug.

**Comment by Yamini234u** [comment]
As checked issue on https://makecode.microbit.org/#editor. The issue is still repro. Please find attachment for the reference

<img width="486" alt="Image" src="https://github.com/user-attachments/assets/3af668db-55e6-4ff1-a9aa-010f5d60bfe4" />

**Comment by abchatra** [comment]
@Yamini234u  please try the /beta editor and not live
